emperor/format.py

```diff
@@ -101,7 +101,7 @@ def format_pcoa_to_js(header, coords, eigvals, pct_var, custom_axes=[],
 
     # write the values for all the ellipses
     js_pcoa_string += '\nvar g_ellipsesDimensions = new Array();\n'
-    if coords_low != None and coords_high != None:
+    if is_empty(coords_low) and is_empty(coords_high):
```

Comment:
You are exactly right, I don't know what I was thinking about, thanks
for catching that!

On (Jan-23-15|14:14), Jorge Cañardo Alastuey wrote:
> > @@ -101,7 +101,7 @@ def format_pcoa_to_js(header, coords, eigvals, pct_var, custom_axes=[],
> >  
> >      # write the values for all the ellipses
> >      js_pcoa_string += '\nvar g_ellipsesDimensions = new Array();\n'
> > -    if coords_low != None and coords_high != None:
> > +    if is_empty(coords_low) and is_empty(coords_high):
> 
> Is this equivalent? When `coords_low = None`, the previous code would result in a False but now it'd return True I think.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/331/files#r23484590